jdbc/README.md

```diff
@@ -54,7 +54,7 @@ Key take aways:
 YCSB has a utility to help create your SQL table. NOTE: It does not support all databases flavors, if it does not work for you, you will have to create your table manually with the schema given above. An example usage of the utility:
 
 ```sh
-java -cp YCSB_HOME/jdbc-binding/lib/jdbc-binding-0.4.0.jar:mysql-connector-java-5.1.37-bin.jar com.yahoo.ycsb.db.JdbcDBCreateTable -P db.properties -n usertable
+java -cp YCSB_HOME/jdbc-binding/lib/jdbc-binding-0.4.0.jar:mysql-connector-java-5.1.37-bin.jar com.yahoo.ycsb.db.voltdb.JdbcDBCreateTable -P db.properties -n usertable
```

Comment:
the JDBC binding should not be moved into a voltdb specific package

Reply:
Fixed in 1424ea4